mobile/src/navigation/course-module/course-module.navigation.tsx

```diff
@@ -18,6 +17,12 @@ const CourseModule: FC = () => {
     isMentor: courses.isMentor,
   }));
 
+  const isShowTaskScreen = isMentor && CourseModuleScreenName.TASK;
+  const filteredScreens = MODULE_TAB_ITEMS.filter(
+    (screen) => screen.name !== isShowTaskScreen,
+  );
+  const isShowTabs = filteredScreens.length === 1;
```

Comment:
```suggestion
  const isTabsShown = Boolean(filteredScreens.length)
```

Reply:
But if we will have 2 and more tabs `Boolean(filteredScreens.length)` return true too.  I understand that for now, we have only 1 or 2 tabs.. but maybe...in the future something change ?)

Reply:
Yes, you are right, then move 1 outside - it should be a constant

Reply:
and also check should be > 1

Reply:
Sorry, don't understand, why should be > 1?

Reply:
If we have more than one screen, than we should render tabs

Reply:
If we will have more than one screen:
 const isTabsShown  = INTERVIEW_TAB_ITEMS.length === 1;
`options={{ tabBarStyle: { display: isTabsShown  ? 'none': 'flex' } }}`
 isTabsShown = false ,  display:'flex'

Reply:
If we have more than one screen, then isTabsShown should be true => display: 'flex', in case we have 1 or 0 screens isTabsShown should be false

Reply:
Maybe I a bit changed logic and we talk about different things)
I have:
`const isHideTabs = filteredScreens.length === 1;`
`options={{ tabBarStyle: { display: isHideTabs ? 'none': 'flex' } }}`
So I have strict equality, and isHideTabs will be true **only** when we have 1 screen, when [screen].length === 1.

`const isHideTabs = [screen, screen].length === 1` = > `isHideTabs: false` => `display: 'flex'`

Reply:
use isTabsShown instead of isTabsHide

Reply:
it's better to check if we should show tabs instead of checking if we should hide them. isTabsShown true means that tabs should be shown